### ISSUE TITLE
Write a wiki?

### ISSUE THREAD

**Issue by luisbocanegra** [luisbocanegra/plasma-panel-colorizer#90]

**Comment by RedSnt** [comment]
It certainly does take some tinkering to get the result one is looking for, I've just been trial and erroring my way to a result, so a wiki would be good. Or at the very least some guidance in the basics like unified widget backgrounds and how preset overrides works. After v4.2.0 "broke" (or fixed I suppose) my theme I spent a good 20 minutes just getting some widget shadows and radius to how it was because I didn't remember how I did it originally 😅

**Comment by luisbocanegra** [comment]
Yeah there are way too many things that even I have found myself does it already do X only to inspect the code and realize it already does!

Documenting and explaining all the things it can do and how to use them is a must at this point!
